mesonbuild/mlog.py

```diff
@@ -18,7 +18,8 @@
 information about Meson runs. Some output goes to screen,
 some to logging dir and some goes to both."""
 
-colorize_console = platform.system().lower() != 'windows' and os.isatty(sys.stdout.fileno())
+colorize_console = platform.system().lower() != 'windows' and os.isatty(sys.stdout.fileno()) and \
+                   os.environ.get('TERM') != 'dumb'
```

Comment:
__[flake8]__
* [E126] continuation line over-indented for hanging indent

<!-- Comment by [SideCI](https://sideci.com) -->